app/models/scholarship_application.rb

```diff
@@ -0,0 +1,15 @@
+class ScholarshipApplication < ApplicationRecord
+  validate :user_is_verified
+
+  belongs_to :user
+  belongs_to :scholarship
+
+  delegate :name, to: :user
+  delegate :email, to: :user
+
+  def user_is_verified
+    unless self.user.verified
```

Comment:
Interestingly enough delegation throws an error when using `?`, so I'll leave it out.